.github/workflows/workflows_scans.yml

```diff
@@ -0,0 +1,37 @@
+name: GitHub Actions Workflows Scans
+on:
+  workflow_dispatch: {}
+  push:
+    paths:
```

Comment:
Is it going to be a required check? If so, we need to get rid of "paths" and either execute it on all changes (since the check is lightweight) or handle its execution via smart CI & overall status check

Reply:
No, we won't make it a required check